sdk/keyvault/keyvault/src/secrets/secretsModels.ts

```diff
@@ -0,0 +1,197 @@
+import * as msRest from "@azure/ms-rest-js";
+
+export interface Secret extends SecretAttributes {
+  /**
+   * @member {string} [value] The secret value.
+   */
+  value?: string;
+}
+
+export interface SecretAttributes extends ParsedKeyVaultEntityIdentifier {
+  /**
+   * @member {string} [id] The secret id.
+   */
+  id?: string;

```

Comment:
Should the id property be on the ParsedKeyVaultEntityIdentifier?

Reply:
In coordination with @jonathandturner, let's address this through this issue: https://github.com/Azure/azure-sdk-for-js/issues/3030